prototype2/client/core/network/src/system.rs

```diff
@@ -67,7 +71,11 @@ impl AdapterSystem {
 impl specs::System<Delta> for AdapterSystem {
   fn run(&mut self, arg: specs::RunArg, _: Delta) {
     let (mut outbound_events, mut inbound_events) = arg.fetch(|w| {
-      (w.write_resource::<Vec<ClientNetworkEvent>>(), w.write_resource::<Vec<ServerNetworkEvent>>())
+      let mut inbound_events = Vec::new();
+      let mut sub = w.read_subscriber(client_event_sub_token).pull();
+      let puub = w.read_publisher(server_event_pub_token);
```

Comment:
This was "puub" because "pub" is a reserved word.

Its dead now though so meh.